crates/astria-conductor/src/block_verifier.rs

```diff
@@ -75,7 +78,7 @@ impl BlockVerifier {
         );
         let validator_set = self
             .sequencer_client
-            .get_validator_set(height)
+            .validators(height - 1, tendermint_rpc::Paging::Default)
```

Comment:
why is this height - 1? change back?

Reply:
@itamarreif this is changed from `height` to `height - 1`, whereas the other one is probably correct.